### PR TITLE
Add genesis.yaml and instructions for how to use it

### PR DESCRIPTION
**Pull request by Anviking**
# Issue Number

# Overview

- [x] Added `genesis.yaml` which we need to **re**-generate the genesis block (which we already store in the repo)
- [x] Added instructions for how to do that + how to launch jormungandr and generate keys with `jcli`

# Comments

Better than nothing I believe.

### CODE REVIEW

**Comment by Anviking** [comment]
Btw, I'm not sure if this output is good
```
jormungandr --genesis-block block-0.bin --config node.config  --secret secret.yaml
May 22 17:33:26.547 INFO storing blockchain in '"/tmp/cardano/storage/blocks.sqlite"'
May 22 17:33:26.553 WARN no gRPC peers specified, skipping bootstrap
May 22 17:33:26.553 INFO starting task: transaction
May 22 17:33:26.553 INFO starting task: client-query
thread 'network' panicked at 'only support the full nodes for now', src/libcore/option.rs:1038:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
May 22 17:33:26.553 INFO starting task: network
May 22 17:33:26.554 INFO our node id: 300798503511280805734040594147789195854
May 22 17:33:26.554 INFO preparing, task: leadership
May 22 17:33:26.554 INFO starting, task: leadership
May 22 17:33:26.554 INFO starting, task: End Of Epoch Reminder, task: leadership
```

**Comment by akegalj** [comment]
> Btw, I'm not sure if this output is good

```
thread 'network' panicked at 'only support the full nodes for now', src/libcore/option.rs:1038:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```
have you tried to run with `RUST_BACKTRACE=1` ?

Out of the blue, I think it should be ok as jormungandr is still wip so probably its not complete implementation yet. 

The error seems to come from https://github.com/input-output-hk/jormungandr/blob/438179d5d769ea0e4c02613f703f088111e880a9/src/network/mod.rs#L80 . Seems like that will be the case if `Configuration` doesn't have  `public_address` defined https://github.com/input-output-hk/jormungandr/blob/438179d5d769ea0e4c02613f703f088111e880a9/src/settings/start/network.rs#L46 . 

Nicolas Di Prima:

> We need to improve this.
> This is pending because we want to make the _public network_ optional for the self node release. People who are just testing don’t necessarily want the _node to node_ protocol enabled

Ante:

> how does node operate if `public_network` is not defined? I guess if it is defined it will connect to another peer in the network. If it is not defined, it will be stand alone node right?
so this bit is still wip and it is ok to see such output

Nicolas Di Prima:

> yes, stand alone node. We are still not sure about what we want to do
> I think we need to make it none optional from the config
`src/setting/start/network.rs` I believe

In this stage of development it is expected to see such output

File: lib/jormungandr/test/data/jormungandr/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+You will need `jcli` and `jormungandr` from https://github.com/input-output-hk/jormungand://github.com/input-output-hk/jormungandr
```
Comment (akegalj) [inline lib/jormungandr/test/data/jormungandr/README.md]: this is a broken link, you pasted it twice concatinated

Comment (akegalj) [reply lib/jormungandr/test/data/jormungandr/README.md]: should be https://github.com/input-output-hk/jormungandr

File: lib/jormungandr/test/data/jormungandr/genesis.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# The Blockchain Configuration defines the different settings
+# of the blockchain that cannot be changed once the blockchain
+# is started.
+blockchain_configuration:
+  peer_2_peer: true
```
Comment (akegalj) [inline lib/jormungandr/test/data/jormungandr/genesis.yaml:5]: I guess in this stage we won't have `peer_2_peer` enabled, but only standalone node

I am still not sure in which state rust node is atm but according to conversation with Nicolas https://github.com/input-output-hk/cardano-wallet/pull/305#issuecomment-495110512 I would assume we should either:
 * set `peer_2_peer` to false and run as a standalone node
 * set `peer_2_peer` to true and connect to the network - but according to what Nicolas said this requires some node configuration to be changed

Also - don't take my word for it as I am just starting to use the node

File: lib/jormungandr/test/data/jormungandr/genesis.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  # of 2 blocks
+  slot_duration: 15
+
+  # The number of blocks (*10) per epoch
+  epoch_stability_depth: 10
+
+  # A list of Ed25519 Extended PublicKey that represents the
+  # BFT leaders encoded as bech32. The order in the list matters.
+  consensus_leader_ids:
+    - ed25519e_pk1xznffwqdhw3drw9yk4t99vpajcc4epq5kp205um5gkkz625xt3mqn4z3mx
```
Comment (akegalj) [inline lib/jormungandr/test/data/jormungandr/genesis.yaml:37]: I wasn't aware we have to setup initial leader list in bft - hm

File: lib/jormungandr/test/data/jormungandr/genesis.yaml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+  # The fee calculations settings
+  #
+  # fee(num_bytes, has_certificate) = constant + num_bytes * coefficient + has_certificate * certificate
+  linear_fee:
+    constant: 0
+    coefficient: 0
+    certificate: 0
+
+  kes_update_speed: 16
```
Comment (akegalj) [inline lib/jormungandr/test/data/jormungandr/genesis.yaml:65]: `kes` stands for?

**Review by akegalj — COMMENTED** [review]
looks good so far. Some minor comments to possibly address

File: lib/jormungandr/test/data/jormungandr/genesis.yaml (hunk 4 above)
Comment (KtorZ) [reply lib/jormungandr/test/data/jormungandr/genesis.yaml:65]: :man_shrugging:

File: lib/jormungandr/test/data/jormungandr/README.md (hunk 1 above)
Comment (KtorZ) [reply lib/jormungandr/test/data/jormungandr/README.md]: Fixed.

**Review by KtorZ — APPROVED** [review]

**Comment by Anviking** [comment]
Thanks @akegalj @KtorZ

File: lib/jormungandr/test/data/jormungandr/genesis.yaml (hunk 2 above)
Comment (Anviking) [reply lib/jormungandr/test/data/jormungandr/genesis.yaml:5]: Hm

**Pull request merged**